libs/pilight/protocols/433.92/tfa303221.c

```diff
@@ -198,7 +198,7 @@ static void parseCode(void) {
 		char *result = malloc(tfa303221->rawlen * 5);
 		char *where = result;
 		for (size_t i = 0; i < tfa303221->rawlen; i++) {
-			size_t printed = sprintf(where, "%d ", tfa303221->raw[i]);
+			size_t printed = snprintf(where, "%d ", tfa303221->raw[i]);
```

Comment:
Did you compile after all these changes? This doesn't compile like this.

Reply:
okay, this should be fixed now